### PR TITLE
feat: support `command + k` shortcuts insert []()

### PR DESCRIPTION
**Pull request by ChasLui**
image

image

### CODE REVIEW

File: web/src/components/MemoEditor.tsx [hunk 1]
```diff
@@ -121,6 +121,15 @@ const MemoEditor = () => {
         editorRef.current.insertText("", "`", "`");
         return;
       }
+      if (event.key === "k") {
+        event.preventDefault();
+        const selectedContent = editorRef.current.getSelectedContent();
+        editorRef.current.insertText("", "[", "]()");
```
Comment (boojack) [inline web/src/components/MemoEditor.tsx:127]:
```suggestion
        editorRef.current.insertText("", "[", "](url)");
```
image

Comment (ChasLui) [reply web/src/components/MemoEditor.tsx:127]: When selecting a part of the content should be `[select range text](|)`?

Comment (boojack) [reply web/src/components/MemoEditor.tsx:127]: Yeah, I think it should be `[selected text](url)`. You can checkout the behavir in this GitHub editor.

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**